### ISSUE TITLE
How could NaCl molecules form triple bond?

### ISSUE THREAD

**Issue by s4il3sh** [ubsuny/23-Homework7G4#36]
![image](https://github.com/ubsuny/23-Homework7G2/assets/144289804/4e1ed8ef-6f44-4fbd-98b5-88a749a23b4d)

As shown in the picture, it seems like Cl and Na ions have formed three bonds which is not correct. What is the correct interpretation?

**Comment by Pranjal-Srivastava-2023** [comment]
How did you figure out the number of bonds from this figure?

**Comment by WildJimmy** [comment]
I think it's just the weird 3d geometry making it hard to tell and not actually triple bonds, but this one is certainly hard to interpret

**Comment by s4il3sh** [comment]
> How did you figure out the number of bonds from this figure?

You can see the central planar atoms. It has 3 bonds in every atoms.

**Comment by s4il3sh** [comment]
> I think it's just the weird 3d geometry making it hard to tell and not actually triple bonds, but this one is certainly hard to interpret

@WildJimmy it seems like an illusion. Something we are missing to interpret this geometry. Maybe we need to see it from different angles.

**Comment by reshnashrestha** [comment]
Yes it is very tricky and the geometry is making it tougher to understand
